api/src/main/java/org/openmrs/api/PersonService.java

```diff
@@ -639,6 +649,7 @@
 	 * @return Person person with given internal identifier
 	 * @throws APIException
 	 * @should return null when no person has the given id
+	 *  @should return null when personId is null
```

Comment:
remove indentation.

did you configure your IDE as described here
https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE
and ran the formatter (either via Eclipse save action or manually) ?